crates/libcontainer/src/process/container_main_process.rs

```diff
@@ -229,6 +237,62 @@ fn setup_mapping(config: &UserNamespaceConfig, pid: Pid) -> Result<()> {
     Ok(())
 }
 
+/// setup_network_device sets up and initializes any defined network interface inside the container.
+fn setup_network_device(
```

Comment:
Could you please be more specific? As far as I know, this function is responsible for moving network devices.